packages/plans-grid/package.json

```diff
@@ -0,0 +1,50 @@
+{
+	"name": "@automattic/plans-grid",
+	"version": "1.0.0-alpha.0",
+	"description": "Plans Grid UI component",
```

Comment:
Might be good to clarify here and in readme that this is WordPress.com plans, just in case anyone comes looking in few years to re-use. 😁